nf-test.config

```diff
@@ -2,5 +2,5 @@ config {
     testsDir "tests"
     workDir ".nf-test"
     configFile "nextflow.config"
-    profile ""
+    profile "test,docker"
```

Comment:
I really think we should leave this out, docker being hard-coded hasn't been the common nf-core practice. I guess it can be overridden, but I just feel like this is going to be a foot gun that we'll hear about in slack.